### PR TITLE
Deprecate scripts and edit msg for user to use the designated command…

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by bakatzir** [comment]
@kirbles19 can you review the change of descriptions?

File: Scripts/script-CheckFilesWildfirePy.yml [hunk 1]
```diff
@@ -46,17 +46,20 @@ tags:
 - server
 - threat-intel
 - wildfire
-comment: Get reputation for attachments using WildFire and if the file is not known
-  upload it to WildFire, wait 15 min and get reputation again
+comment: Deprecated, use "WildFire - Detonate File" playbook instead
```
Comment (kirbles19) [inline Scripts/script-CheckFilesWildfirePy.yml]: Deprecated. Use the "Wildfire - Detonate File" playbook instead.

File: Scripts/script-CheckFilesWildfirePy.yml [hunk 2]
```diff
@@ -46,17 +46,20 @@ tags:
...
 scripttarget: 0
+deprecated: true
 dependson:
   must:
   - wildfire-upload
   - wildfire-report
 timeout: 0s
+tests:
+- No tests
+releaseNotes: Deprecated, use "WildFire - Detonate File" playbook instead
```
Comment (kirbles19) [inline Scripts/script-CheckFilesWildfirePy.yml]: Deprecated. Use the "Wildfire - Detonate File" playbook instead.

File: Scripts/script-PanoramaMove.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ type: javascript
 tags:
 - panorama
 - firewall
-comment: Move panorama rules
+comment: Deprecated, use panorama-move-rule instead
```
Comment (kirbles19) [inline Scripts/script-PanoramaMove.yml]: Deprecated. Use the "panorama-move-rule" command instead.

File: Scripts/script-PanoramaMove.yml [hunk 4]
```diff
@@ -42,4 +42,5 @@ dependson:
   - panorama
 timeout: 0s
 tests:
-- palo_alto_firewall_test_pb
+- No tests
+releaseNotes: Deprecated, use panorama-move-rule instead
```
Comment (kirbles19) [inline Scripts/script-PanoramaMove.yml]: Deprecated. Use the "panorama-move-rule" command instead.

File: Scripts/script-PanoramaPcaps.yml [hunk 5]
```diff
@@ -38,7 +38,7 @@ type: javascript
 tags:
 - panorama
 - firewall
-comment: Get panorama pcaps
+comment: Deprecated, use panorama-get-pcap instead
```
Comment (kirbles19) [inline Scripts/script-PanoramaPcaps.yml]: Deprecated. Use the "panorama-get-pcap" command instead.

File: Scripts/script-PanoramaPcaps.yml [hunk 6]
```diff
@@ -66,4 +66,5 @@ dependson:
   - panorama
 timeout: 0s
 tests:
-- palo_alto_firewall_test_pb
+- No tests
+releaseNotes: Deprecated, use panorama-get-pcap instead
```
Comment (kirbles19) [inline Scripts/script-PanoramaPcaps.yml]: Deprecated. Use the "panorama-get-pcap" command instead.

File: Scripts/script-WildfireReport.yml [hunk 7]
```diff
@@ -39,7 +39,7 @@ tags:
 - enhancement
 - wildfire
 - sandbox
-comment: Use WildFire file analysis and parse result
+comment: Deprecated, use wildfire-report instead
```
Comment (kirbles19) [inline Scripts/script-WildfireReport.yml]: Deprecated. Use the "wildfire-report" command instead.

File: Scripts/script-WildfireReport.yml [hunk 8]
```diff
@@ -52,3 +52,6 @@ dependson:
   must:
   - wildfire-report
 timeout: 0s
+tests:
+- No tests
+releaseNotes: Deprecated, use wildfire-report instead
```
Comment (kirbles19) [inline Scripts/script-WildfireReport.yml]: Deprecated. Use the "wildfire-report" command instead.

File: Scripts/script-WildfireUpload.yml [hunk 9]
```diff
@@ -27,9 +27,7 @@ type: javascript
 tags:
 - wildfire
 - sandbox
-comment: |-
-  Upload file to WildFire for analysis.
-  Example usage: !WildfireUpload upload=2@3
+comment: Deprecated, use wildfire-upload instead
```
Comment (kirbles19) [inline Scripts/script-WildfireUpload.yml]: Deprecated. Use the "wildfire-upload" command instead.

File: Scripts/script-WildfireUpload.yml [hunk 10]
```diff
@@ -41,3 +39,6 @@ dependson:
   must:
   - wildfire-upload
 timeout: 0s
+tests:
+- No tests
+releaseNotes: Deprecated, use wildfire-upload instead
```
Comment (kirbles19) [inline Scripts/script-WildfireUpload.yml]: Deprecated. Use the "wildfire-upload" command instead.

**Pull request merged**